docker/docker/src/test/java/org/arquillian/cube/docker/impl/client/BeforeStopContainerObserverTest.java

```diff
@@ -32,6 +35,7 @@
 import static org.mockito.Mockito.verify;
 
 @RunWith(MockitoJUnitRunner.class)
+@Category({RequiresDocker.class, RequiresDockerMachine.class})
```

Comment:
Do we really need these categories if we are running mockito-powered tests?

Reply:
Now if we decided to just run all docker related tests (unit and functional), currently there is no way, if we run tests using category, it will execute all unit tests (docker, k8s, openshift) and functional tests having `@Category(RequiresDocker.class)`, as for all unit tests we don't have any category.

So to run particular env(docker/k8s) unit tests I applied `@Category` for all unit tests

Reply:
With this also you can run tests on host having docker or host having docker-machine without commenting anything if we fix how to get docker machine installed on host?

Reply:
I understand this reasoning, but this effectively means that we have to have a category always defined for every test or run the build without `groups` defined. I am not sure if that's intuitive.

In addition, it can be overlooked (unless we bring some check during the build to enforce having categories on each test), so if we don't have a build on our CI without `groups` defined those tests will never be run.